raisinbread/instruments/instrument_sql/mri_parameter_form.sql

```diff
@@ -26,6 +26,27 @@ CREATE TABLE `mri_parameter_form` (
 `2d_flair_Scan_Done` enum('yes','partial','no') default NULL,
 `2d_flair_number_attempts` varchar(255) default NULL,
 `2d_flair_comments` text default NULL,
+`fieldmapBOLD_Scan_Done` enum('yes','partial','no') DEFAULT NULL,
```

Comment:
Did you verify that these match:
1. the mri_scan_types table
2. the instrument itself used by raisinbread?

Reply:
1. yes but whoever is codereviewing should confirm that
2. no these fields will not show in the instrument. I am doing this simply to bugfix modules that are too dump to handle a missing field.

future changes should include. 
- adding these fields to the instrument front end
- improving quality control to be more elegant in handling the missing columns